gss-admin-client.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import requests
+import logging
+
+# GSS admin OpenAPI definition https://<your-gss-site.esa.int>/gss-admin-api/v3/api-docs
+ENTITIES = [
+    "datastores/hfs", "consumers", "producers", "quotas", 
+    "metadatastores/solr", "datastores/timebased", "swiftcredentials", 
+    "jobs", "datastores/swiftgroup", "datastores/swift"
+]
+BASE_URL = "https://collgs.cesnet.cz/gss-admin-api"
+
+def load_json_file(file_path):
+    with open(file_path, "r") as f:
+        return json.load(f)
+
+def get_client():
+    return {
+        "base_url": BASE_URL,
+        #"auth": ("your_username", "your_password"),
+        "auth": None, # use .netrc or no auth
+        "headers": {"Content-Type": "application/json"},
+    }
+
+def print_entry(entry, pretty):
+    if args.pretty:
+        print(json.dumps(entry, sort_keys=True, indent=4))
+    else:
+        print(json.dumps(entry))
+
+# The processes and displays JSON responses so we can save&edit later
+def print_entries(response, entity, pretty):
+    try:
+        data = response.json()
+        if isinstance(data, dict) and entity in data:
+            for key, entries in data.items():
+                if isinstance(entries, list):
+                    for entry in entries:
+                        print_entry(entry, pretty)
+        elif isinstance(data, list):
+            for entry in data:
+                print_entry(entry, pretty)
+        else:
+            print_entry(data, pretty)
+    except json.JSONDecodeError:
+        logging.error("Invalid JSON response")
+
+def handle_request(client, entity, action, entity_name=None, data=None, pretty=False):
+    url = f"{client['base_url']}/{entity}"
+    
+    if action == "get":
+        if entity_name:
+            url = f"{url}/{entity_name}"
+        response = requests.get(url, auth=client['auth'], headers=client['headers'])
```

Comment:
```suggestion
        response = requests.get(url)
```
related to requests.session